build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'idea'
```

Comment:
What does the `idea` plugin do? 
Can we do without it?

Reply:
Probably yes - it just adds task to auto-configure the project for IntelliJ IDEA

But i wouldn't be surprised if it's capable of configuring itself well enough even without it, i'll give that a test
[Idea plugin in the docs](https://docs.gradle.org/current/userguide/idea_plugin.html)